nucypher/storage/policy.py

```diff
@@ -0,0 +1,105 @@
+import os
+from abc import ABC, abstractmethod
+from typing import Iterator, List, Union
+
+from nucypher.config.constants import DEFAULT_CONFIG_ROOT
+from nucypher.policy.collections import PolicyCredential
+
+
+class PolicyCredentialStorage(ABC):
```

Comment:
:+1:

Reply:
Not in this PR, but it might be worth thinking about how we can establish a common base class for NodeStorage and PolicyStorage so that we don't have repeated code for the two - may or may not be feasible.